ms2deepscore/data_generators.py

```diff
@@ -476,6 +474,110 @@ def __len__(self):
         This is expected behavior, with the shuffling this is OK.
         """
         return int(self.settings["num_turns"]) * int(np.floor(len(self.reference_scores_df) / self.settings["batch_size"]))
+    def _spectrum_pair_generator(self, batch_index: int) -> Iterator[SpectrumPair]:
```

Comment:
The _spectrum_pair_generator was not changed/added, this is just Github being confused, since the next class has very similar code